File: clientapp/src/app/models/joborder.ts
export interface Joborder
{
    id: number;
    namaKlien: string;
    nomorSurat: string;
    tanggalSurat: Date;
    idKoordinator: number;
    koordinator: string;
    notes: string;
    currentExpense: number;
    status: JobStatus;
    createTime: Date;
    closeTime: Date;
}
enum JobStatus { 
    Open,
    Close
}

File: serverside/Core/Model/JobOrder.cs
using System;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace serverside.Core.Models
{
    public class Joborder
    {
        public int Id { get; set; }
        public string NamaKlien { get; set; }
        public string NomorSurat { get; set; }
        public DateTime TanggalSurat { get; set; }
        [ForeignKey("Koordinator")]
        public int IdKoordinator{ get; set; }
        public User Koordinator {get; set;}
        public string Notes { get; set; }
        public int CurrentExpense { get; set; }
        public JobStatus Status { get; set; }
        //system
        public DateTime CreateTime { get; set; }
        public DateTime CloseTime { get; set; }

    }
}